### DISCUSSION TITLE
[@types/ws] v8.5.5 Breaking changes. Server is not generic

### DISCUSSION THREAD

**Discussion by Mehayes411** [DefinitelyTyped/DefinitelyTyped#65731] [Issues with a @types package]
Breaking changes is 8.5.5
Used by:
└─┬ @angular-devkit/build-angular@14.2.6
  └─┬ webpack-dev-server@4.11.0
    └── @types/ws@8.5.5

Error: node_modules/@types/ws/index.d.ts:328:34 - error TS2315: Type 'Server' is not generic.
328         server?: HTTPServer<V> | HTTPSServer<V> | undefined;

[a comment by an automated account is not shown]

**Comment by vansergen** [comment]
Should be fixed in [v4.15.1](https://github.com/webpack/webpack-dev-server/releases/tag/v4.15.1)
